hivemind_redis_database/__init__.py

```
@@ -9,22 +9,220 @@
+
+    def _check_redisearch_availability(self) -> bool:
+        """
+        Check if RediSearch module is available on the Redis server.
+
+        Returns:
+            True if RediSearch is available, False otherwise.
+        """
+        try:
+            # Try to execute a simple RediSearch command
+            self.redis.ft("test_index").info()
+            return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)